qcore/constants.py

```diff
@@ -294,6 +294,8 @@ class FaultParams(Enum):
     vel_mod_dir = "vel_mod_dir"
     stat_coords = "stat_coords"
     FD_STATLIST = "FD_STATLIST"
+    site_vm_dir = "site_vm_dir"
```

Comment:
May have to update this config name too?

Reply:
Ah, forgot to push. Nice catch.